components/pages/workPageLayout/WorkPageButtons.tsx

```diff
@@ -68,45 +69,57 @@ const WorkPageButtons = ({ workId, selectedManifestation }: WorkPageButtonsProps
           )}
           <Button
             ariaLabel={isLoaned ? "Læs e-bog" : "Lån e-bog"}
+            size={"lg"}
             className="w-full lg:max-w-80 lg:min-w-72"
             theme={"primary"}
-            disabled={isLoanButtonDisabled}>
+            disabled={isLoanButtonDisabled}
+            onClick={() => !isLoaned && setIsConfirmDialogOpen(!isConfirmDialogOpen)}>
             {isLoaned ? "Læs e-bog" : "Lån e-bog"}
           </Button>
         </>
       )}
       {isManifestationAudioBook(selectedManifestation) && (
         <>
-          <Button
-            ariaLabel="Prøv lydbog"
-            disabled={!!!identifier}
-            className="mb-grid-gap-half w-full lg:max-w-80 lg:min-w-72"
-            onClick={() => setIsPlayerOpen(!isPlayerOpen)}>
-            Prøv lydbog
-          </Button>
+          {!isLoaned && (
+            <Button
+              ariaLabel="Prøv lydbog"
+              size={"lg"}
+              disabled={!!!identifier}
```

Comment:
It is a matter of personal preference but I think `!!!!` is unreadable. To me `!Boolean(identifier)` is more clear.

Reply:
I can change that